src/librbd/api/Mirror.cc

```diff
@@ -2954,6 +2954,27 @@ int Mirror<I>::group_disable(IoCtx& group_ioctx, const char *group_name,
     return r;
   }
 
+  // removing pending resync request
+  group_header_oid = librbd::util::group_header_name(group_id);
+  std::string value;
+  r = librbd::cls_client::metadata_get(&group_ioctx, group_header_oid,
+                                       RBD_GROUP_RESYNC, &value);
+  if (r < 0 && r != -ENOENT) {
+    lderr(cct) << "failed reading metadata: " << RBD_GROUP_RESYNC << " : "
+               << cpp_strerror(r) << dendl;
+    return r;
+  } else if (r == 0) {
+    ldout(cct, 5) << "found resync group request, clearing it"
```

Comment:
Nit:
I don't think this log is important to be noted or logged, if you have a justification please add it, in anyway we will to adjust the loglevel to 10 if not 20.